centreon/tests/php/mock/CentreonDBStatement.php

```
@@ -0,0 +1,253 @@
+            $this->params[$param - 1] = $value;
+        } else {
+            $this->params[$param] = $value;
+        }
+
+        return true;
+    }
+
+    /**
+     * Execute statement
+     */
+    public function execute($params = null): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$params'. (Unused Code Rules)

(UnusedFormalParameter)